Python/setup.py

```diff
@@ -41,29 +41,36 @@
 import os
 import setuptools
 
+
+def read_requirements():
+    requirements = []
+    with open('./requirements.txt', 'r') as file:
+        for line in file:
+            line = str(line)  # guard against str exceptions
```

Comment:
Not necessary. You opened it at 'r' which is text in 3 but you can make explicit and do 'rt'. All string exceptions will then happen at the open.

Reply:
Thanks, I missed that. Meant to do "rt".

Reply:
This is addressed, will be in next commit.